DataManagementSystem/Agent/RequestOperations/DMSRequestOperationsBase.py

```diff
@@ -13,19 +13,33 @@
 from DIRAC.RequestManagementSystem.Client.File                  import File
 from DIRAC.Resources.Storage.StorageElement                     import StorageElement
 from DIRAC.RequestManagementSystem.private.OperationHandlerBase import OperationHandlerBase
-from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers
+from DIRAC.DataManagementSystem.Utilities.DMSHelpers            import DMSHelpers
+from DIRAC.ResourceStatusSystem.Client.ResourceStatus           import ResourceStatus
 
 class DMSRequestOperationsBase( OperationHandlerBase ):
 
   def __init__( self, operation = None, csPath = None ):
     OperationHandlerBase.__init__( self, operation, csPath )
     self.registrationProtocols = DMSHelpers().getRegistrationProtocols()
+    self.rssFlag = ResourceStatus().rssFlag
 
 
   def checkSEsRSS( self, checkSEs = None, access = 'WriteAccess' ):
-    """ check SEs.
+    """ check SEs, returning a list of banned SEs.
         By default, we check the SEs for WriteAccess, but it is configurable
+
+        :param checkSEs: SEs to check, by default the operation target list is used.
+        :param access: The type of access to check for ('WriteAccess' or 'ReadAccess').
+        :type checkSEs: list
```

Comment:
please use `python:list` for the type, otherwise the Dirac.Core.Utilities.List is found instead. (same for rtype)